# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaDecoderStateMachine.h
# Commit: 7d876e0ef72a
# Full Hash: 7d876e0ef72a5543c0c5ad207cde8b115f937036
# Author: alwu <alwu@mozilla.com>
# Date: 2022-10-18 09:48:31
# Regressor Bug: 1262276
# File Overlap Count: 1
# Description:
#   Bug 1262276 - part2 : support seamless video looping. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D159126
# ==============================================================================

diff -r 70bc7e69d03a -r 7d876e0ef72a dom/media/MediaDecoderStateMachine.h
--- a/dom/media/MediaDecoderStateMachine.h	Mon Oct 17 18:28:16 2022 +0000
+++ b/dom/media/MediaDecoderStateMachine.h	Mon Oct 17 18:28:17 2022 +0000
@@ -512,6 +512,7 @@
   // to adjust sample time from clock time to media time.
   void AdjustByLooping(media::TimeUnit& aTime) const;
   Maybe<media::TimeUnit> mAudioDecodedDuration;
+  Maybe<media::TimeUnit> mVideoDecodedDuration;
 
   // Current playback position in the stream in bytes.
   int64_t mPlaybackOffset = 0;
